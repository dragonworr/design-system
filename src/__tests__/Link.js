import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Link } from '..'

describe('Link', () => {
  test('renders', () => {
    const json = renderer.create(<Link>Dummy</Link>).toJSON()
    expect(json).toMatchSnapshot()
  })
})
