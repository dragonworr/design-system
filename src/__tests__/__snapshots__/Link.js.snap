// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders 1`] = `
.c0 {
  text-decoration: none;
  color: #007aff;
}

.c0:hover {
  text-decoration: underline;
}

<a
  className="c0"
  color="blue"
>
  Dummy
</a>
`;
